### PR TITLE
Manish/roger 829

### PR DESCRIPTION
**Pull request by ranjanmanish**

### CODE REVIEW

**Comment by amitbose327** [comment]
Please remove or update the 'roger-mesos-toolsconfig.yml' files under examples and tests directories. They have Roger specific stuff and this repo is a public repo. Also based on the comments in https://github.com/seomoz/roger-cli/pull/268, you might to make the necessary changes. Rest looks good.

**Comment by ranjanmanish** [comment]
The file content under example or test has not changed. Just the format has been updated.  Hence I did not omit/mask any further details.

**Comment by ranjanmanish** [comment]
I see. Okay get it.

File: cli/appconfig.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class AppConfig:
 
     def getRogerEnv(self, config_dir):
         roger_env = None
-        env_file = '{0}/roger-env.json'
+        env_file = '{0}/roger-mesos-toolsconfig.yml'
```
Comment (ankanm) [inline cli/appconfig.py]: `roger-mesos-tools.config`

File: examples/config/roger-mesos-toolsconfig.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+default_environment: dev
```
Comment (ankanm) [inline examples/config/roger-mesos-toolsconfig.yml]: For the endpoints, and registry - Remove all the `moz.com` urls and replace with `example.com` urls and no port numbers. e.g. `chronos.example.com`, `marathon.example.com`, `registry.example.com` and so on...

File: examples/config/roger-mesos-toolsconfig.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+default_github_repo_prefix: git@github.com:seomoz/
```
Comment (ankanm) [inline examples/config/roger-mesos-toolsconfig.yml]: Change `seomoz` to `example`

File: tests/configs/roger-env.json [hunk 4]
```diff
@@ -14,5 +14,5 @@
       "chronos_endpoint": "http://prod.example.com:4400"
```
Comment (ankanm) [inline tests/configs/roger-env.json]: This file shouldn't be needed.

**Comment by amitbose327** [comment]
LGTM

**Comment by ankanm** [comment]
LGTM

**Pull request merged**